CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ and this project adheres to
 
 ## Unreleased
 
+### Added
+
+- A `deprecationDate` property on types `StepRelationshipMetadata` and
```

Comment:
I'm good with using those properties, though I don't really want to use a timestamp. To me, a timestamp means that we are committing to removing these relationships at that time which is far larger of a commitment than I want to promise.

I want some word that indicates that this relationship should no longer be trusted to exist after a certain date. That way we can do what we want with it at our leisure. `sunsetDate` for the most part gets that across, but I can also see someone asking why they have relationships past their `sunsetDate`. `deprecationDate` sidesteps that issue, but it might trick people into thinking we are just going to leave it alone instead of removing them.

Reply:
Thinking about this again again, I like `endOfLifeDate` more than `sunsetDate` as it is a common term.